src/components/Post.jsx

```diff
@@ -184,6 +187,28 @@ function Post(prop) {
     setOpen(true);
     setAnchorEl(null);
   };
+  const handleClickOpenCaption=async()=>{
+    
```

Comment:
extra line, lots of formatting issues. format the whole code please using a formatter